airflow/cli/commands/task_command.py

```diff
@@ -430,20 +434,26 @@ def task_run(args, dag: DAG | None = None) -> TaskReturnCode | None:
     # this should be last thing before running, to reduce likelihood of an open session
     # which can cause trouble if running process in a fork.
     settings.reconfigure_orm(disable_connection_pool=True)
+    log.info("DID RECONFIGURE")
     task_return_code = None
     try:
         if args.interactive:
             task_return_code = _run_task_by_selected_method(args, _dag, ti)
+            log.info("RAN INTERACTIVE")
+            raise Exception("interactive?")
+
         else:
-            with _move_task_handlers_to_root(ti), _redirect_stdout_to_ti_log(ti):
+            with _move_task_handlers_to_root(ti):
```

Comment:
was getting a recursion error but i may have fixed that with the `_logger_name` change